### ISSUE TITLE
File Uploader (v2): Cannot activate the remove button by keyboard only

### ISSUE THREAD

**Issue by stanislavgeorgiev** [IBM/carbon-components-angular#556]
## Detailed description

> Describe in detail the issue you're having.

Our AVT team flagged that the remove button cannot be activated by keyboard, pressing Enter or Space doesn't do anything.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> Is this issue related to a specific component?

File uploader

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

v2@latest

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM ADDI

**Comment by stanislavgeorgiev** [comment]
Would a review be coming up soon?

**Comment by cal-smith** [comment]
Yep, It's up to @zvonimirfras for when though 👍

**Comment by stanislavgeorgiev** [comment]
Can we have this merged in 2x branch too?

**Comment by cal-smith** [comment]
Sure!
